### DISCUSSION TITLE
Is Spring.Net a dead project?

### DISCUSSION THREAD

**Discussion by hfickes** [spring-projects/spring-net#216] [General]
We started using Spring.Net (just the core) for IoC about 10 years ago.  We hadn't updated in quite a few years and looking at it now, it seems that Spring.Net has been on life-support at best.  The version I pulled using nuget is working with an outdated common.logging and log4net.

Is this still an active project?

**Comment by lahma** [comment]
See https://github.com/spring-projects/spring-net/issues/187#issuecomment-891153665

**Reply by hfickes** [reply]
While there is a version from Aug 2021 in github, the project website at https://www.springframework.net/download.html has its most recent version as the one from 10 years ago.  That site probably should be redirected here.

It also looks like the latest Spring.Core.dll isn't signed which prevents some deployment options from .Net Framework.
